src/components/ec-table-sort/ec-table-sort.vue

```diff
@@ -1,34 +1,26 @@
 <template>
-  <div
+  <a
```

Comment:
can this be a `button`?

Reply:
Why should have it as a button?

Reply:
I like this rule of thumb from https://css-tricks.com/a-complete-guide-to-links-and-buttons/

> Are you making a JavaScript-powered clickable action? Use a button

this link doesn't point to any url, let's make it a button